templates/search/featured_search.html

```diff
@@ -31,23 +31,24 @@
         {% endif %}
     </div>
     {% endif %}
-    {% if buckets.researchGuide.result_count > 0 or buckets.insight.result_count > 0 or buckets.blog.result_count > 0 %}
-    <div class="featured-search__interpretive-results">
-        {% if buckets.researchGuide.result_count > 0 %}
-            {% include './blocks/featured-search__interpretive-card.html' with bucket=buckets.researchGuide %}
-        {% endif %}
-
-        {% if buckets.insight.result_count > 0 %}
-            {% include './blocks/featured-search__interpretive-card.html' with bucket=buckets.insight %}
-        {% endif %}
 
-        {% if buckets.blog.result_count > 0 %}
-            {% include './blocks/featured-search__interpretive-card.html' with bucket=buckets.blog %}
-        {% endif %}
-    </div>
+    {% if website_result_count %}
+        <section class="featured-search__website-results">
+            <header class="featured-search__website-results__head">
+                <h2 class="featured-search__website-results__heading">Results from the website</h2>
+                <a href="/search/website?q={{ search_query|iriencode }}" class="featured-search__website_results-link">
```

Comment:
For default search, the link appears as http://127.0.0.1:8000/search/website?q=None, the `search_query|iriencode` does not seem to convert None, looks like `self.query = self.form.cleaned_data.get("q", "")` is not getting its default.

Reply:
@TNA-Allan on investigating, it looks like this is down to the `q` field on the forms having an `empty_value` value set to `None`. I'm just figuring out the best course of action here.

Reply:
Fixed in [3d45b0d](https://github.com/nationalarchives/ds-wagtail/pull/1136/commits/3d45b0de396e9d71dfb8ec85fd9533a8fdb4dbc7)

Reply:
Thanks @ababic